File: src/main/java/kr/co/lunatalk/domain/exhibition/dto/response/ExhibitionCreateResponse.java
package kr.co.lunatalk.domain.exhibition.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;
import kr.co.lunatalk.domain.exhibition.domain.Exhibition;

public record ExhibitionCreateResponse(
	@Schema(description = "생성한 exhibitionId")
	Long exhibitionId
) {
	public static ExhibitionCreateResponse from(Exhibition exhibition) {
		return new ExhibitionCreateResponse(exhibition.getId());
	}
}

File: src/main/kotlin/kr/co/lunatalk/domain/exhibition/dto/response/ExhibitionCreateResponse.kt
package kr.co.lunatalk.domain.exhibition.dto.response

import io.swagger.v3.oas.annotations.media.Schema
import kr.co.lunatalk.domain.exhibition.domain.Exhibition

data class ExhibitionCreateResponse(
    @field:Schema(description = "생성한 exhibitionId")
    val exhibitionId: Long?
) {
    companion object {
        fun from(exhibition: Exhibition): ExhibitionCreateResponse {
            return ExhibitionCreateResponse(exhibition.id)
        }
    }
}
